### PR TITLE
feat(condo): DOMA-5487 The `NewsItemTemplate` model

### PR DESCRIPTION
**Pull request by AleX83Xpert**
- [x] access rules
- [x] tests

### CODE REVIEW

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/news/schema/NewsItemTemplate.test.js [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+            await makeClientWithRegisteredOrganization()
+            residentClient = await makeClientWithResidentUser()
+            anonymousClient = await makeClient()
+            staffClient = await makeClientWithStaffUser()
+            staffClientNoPermission = await makeClientWithStaffUser()
+
+            const [templateCreated] = await createTestNewsItemTemplate(adminClient)
+            dummyNewsItemTemplate = templateCreated
+
+            const [o10n] = await createTestOrganization(adminClient)
```
Comment (sitozzz) [inline apps/condo/domains/news/schema/NewsItemTemplate.test.js:60]: Strange naming for me. Does it make sense?

**Comment by sitozzz** [comment]
I can't find any relations to the original model. So how it will work with `NewsItem`?

File: apps/condo/domains/news/schema/NewsItemTemplate.test.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/schema/NewsItemTemplate.test.js:60]: It's a short name for `o r g a n i z a t i o n`, Like i18n :)

**Comment by AleX83Xpert** [comment]
> I can't find any relations to the original model. So how it will work with `NewsItem`?

No such links are needed. These templates will use at the frontend to quickly create content for news items.

**Pull request merged**